docs/src/content/docs/es/guides/css-and-tailwind.mdx

```diff
@@ -0,0 +1,288 @@
+---
+title: CSS y Estilos
+description: Aprende a darle estilo a tu sitio Starlight con CSS personalizado o integrado con Tailwind CSS.
+---
+
+Puedes darle estilo a tu sitio Starlight con archivos CSS personalizados o usar el plugin Starlight Tailwind.
+
+## Estilos CSS personalizados
+
+Personaliza los estilos aplicados a tu sitio Starlight proporcionando archivos CSS adicionales para modificar o extender los estilos predeterminados de Starlight.
+
+1. Agrega un archivo CSS a tu directorio `src/`.
+   Por ejemplo, podrías establecer un ancho de columna predeterminado más ancho y un tamaño de texto más grande para los títulos de las páginas:
+
+   ```css
+   /* src/styles/custom.css */
+   :root {
+   	--sl-content-width: 50rem;
+   	--sl-text-5xl: 3.5rem;
+   }
+   ```
+
+2. Agrega la ruta de tu archivo CSS a la matriz `customCss` de Starlight en `astro.config.mjs`:
+
+   ```js
+   // astro.config.mjs
+   import { defineConfig } from 'astro/config';
+   import starlight from '@astrojs/starlight';
+
+   export default defineConfig({
+   	integrations: [
+   		starlight({
+   			title: 'Docs With Custom CSS',
+   			customCss: [
+   				// Ruta relativa a tu archivo CSS personalizado
+   				'./src/styles/custom.css',
+   			],
+   		}),
+   	],
+   });
+   ```
+
+Puede ver todas las propiedades CSS personalizadas utilizadas por Starlight que puede establecer para personalizar su sitio en el archivo [`props.css` en GitHub](https://github.com/withastro/starlight/blob/main/packages/starlight/style/props.css).
+
+## Tailwind CSS
+
+El soporte de CSS de Tailwind en proyectos de Astro es proporcionado por la [integración Astro Tailwind](https://docs.astro.build/en/guides/integrations-guide/tailwind/).
+Starlight proporciona un complemento de Tailwind para ayudar a configurar Tailwind para que sea compatible con los estilos de Starlight.
+
+El plugin Starlight Tailwind aplica la siguiente configuración:
+
+- Configura las vairantes de Tailwind `dark:` para que funcionen con el modo oscuro de Starlight.
+- Usa los [colores y fuentes del tema](#estilando-starlight-con-tailwind) de Tailwind en la UI de Starlight.
+- Desactiva la [comprobación preliminar](https://tailwindcss.com/docs/preflight) de Tailwind mientras restaura selectivamente las partes esenciales de la comprobación preliminar requeridas para las clases de utilidad de borde de Tailwind.
```

Comment:
I think it could be important to keep a mention of the “Preflight” name here as Tailwind users might be familiar with what that is. Maybe something like this? Feel free to rework it as you think sounds best!

```suggestion
- Desactiva la [comprobación preliminar “Preflight”](https://tailwindcss.com/docs/preflight) de Tailwind mientras restaura selectivamente las partes esenciales de la comprobación preliminar requeridas para las clases de utilidad de borde de Tailwind.
```